suzuki/reactions_suzuki.py

```diff
@@ -0,0 +1,18 @@
+from chython import smiles
+from chython.reactor import reactions
+from itertools import product
+with open('reag1.txt') as f:
+    for line in f:
+        a = line.strip()
```

Comment:
Можно поменять на что-то такое, более наглядное
```suggestion
        reagent_1 = line.strip()
```